testscpp/tests/test_presolve.cpp

```diff
@@ -14,7 +14,7 @@
 
 #include "catch2/catch.hpp"
 #include "dimod/constrained_quadratic_model.h"
-#include "dwave/presolve.h"
+#include "dwave/presolve.hpp"
```

Comment:
I know that the IDE complains, but technically this is a backwards compatibility break so I think we're stuck with `.h` until `0.6.0`. We could do an alias thing, but IMO that's overkill.